src/CalcManager/win_data_types_cross_platform.h

```diff
@@ -7,14 +7,14 @@ typedef int32_t INT;
 typedef uint8_t CHAR;
 typedef int32_t LONG;
 typedef uint8_t BYTE;
-typedef unsigned int UINT_PTR;
+typedef uintptr_t UINT_PTR;
 typedef unsigned long ULONG_PTR;
-typedef unsigned int ULONG32;
-typedef unsigned long DWORD;
-typedef unsigned long ULONG;
-typedef unsigned short USHORT;
+typedef uint32_t ULONG32;
+typedef uint32_t DWORD;
+typedef uint32_t ULONG;
+typedef uint16_t USHORT;
 typedef wchar_t WCHAR;
-typedef unsigned short WORD;
+typedef uint16_t WORD;
```

Comment:
These values are based on https://docs.microsoft.com/en-us/windows/desktop/WinProg/windows-data-types

Reply:
I know, I used the same page (and filed feedback regarding the errors there)

Reply:
This whole block should not be needed anymore once https://github.com/Microsoft/calculator/pull/212 lands